cogs/welcome/welcome.py

```diff
@@ -318,6 +339,37 @@ async def logServerLeave(self, leaveUser: discord.Member):
                     leaveUser.id,
                 )
 
+    async def generateRandWelcomeImg(self, user):
+        """creates an image for the specific player using their avatar and an image from the random image pool, then returns it"""
+        base = Image.open(os.path.join(self.img_dir, random.choice(os.listdir(self.img_dir))))
+        mask = Image.open(os.path.join(os.path.dirname(os.path.realpath(__file__)), "MASK.png"))
+        border_overlay = Image.open(
+            os.path.join(os.path.dirname(os.path.realpath(__file__)), "BORDER.png")
+        )
+        border_overlay_mask = Image.open(
+            os.path.join(os.path.dirname(os.path.realpath(__file__)), "BORDER_mask.png")
+        )
```

Comment:
[Consider using context managers or else these should be closed later on.](https://pillow.readthedocs.io/en/stable/reference/open_files.html#proposed-file-handling)
```suggestion
        with Image.open(
            os.path.join(os.path.dirname(os.path.realpath(__file__)), "BORDER.png")
        ) as border_overlay:
            with Image.open(
                os.path.join(os.path.dirname(os.path.realpath(__file__)), "BORDER_mask.png")
            ) as border_overlay_mask:
```